DABI.js

```diff
@@ -0,0 +1,677 @@
+{
+    "translatorID": "5cf8bb21-e350-444f-b9b4-f46d9fab7827",
+    "label": "DABI",
+    "creator": "Jens Mittelbach",
+    "target": "dabi.ib.hu-berlin\\.de/.*?",
+    "minVersion": "1.0",
+    "maxVersion": "",
+    "priority": 100,
+    "inRepository": true,
+    "translatorType": 4,
+    "browserSupport": "g",
+    "lastUpdated": "2015-12-15 18:59:56"
+}
+
+/* FW LINE 57:6869c32952b1 */
```

Comment:
I think this happened because of jsbeautify, but we definitely do not want to expand the framework code. I would suggest removing all of the code from this line down to the `function detectWeb...` and then re-saving the translator in Scaffold with the "Use translator framework" checkbox ticked (Metadata tab)